lms/static/scripts/frontend_apps/components/StudentSelector.js

```diff
@@ -86,30 +93,63 @@ export default function StudentSelector({
         >
           {options}
         </select>
-        <Icon classes="StudentsSelector__students-icon" name="caretDown" />{' '}
+        <Icon
+          classes="absolute top-0.5 right-3 pointer-events-none text-grey-4"
```

Comment:
If you're curious, the `text-grey-4` class is available because of the upstream (`frontend-shared`) tailwind config, which defines that color.